lightly/openapi_generated/swagger_client/api/samples_api.py

```diff
@@ -461,6 +461,109 @@ def get_sample_image_write_url_by_id_with_http_info(self, dataset_id, sample_id,
+        >>> thread = api.get_sample_image_write_urls_by_id_with_http_info(dataset_id, sample_id, async_req=True)
+        >>> result = thread.get()
+
+        :param async_req bool
+        :param MongoObjectID dataset_id: ObjectId of the dataset (required)
+        :param MongoObjectID sample_id: ObjectId of the sample (required)
+        :return: SampleWriteUrls
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+
+        all_params = ['dataset_id', 'sample_id']  # noqa: E501
+        all_params.append('async_req')
+        all_params.append('_return_http_data_only')
+        all_params.append('_preload_content')
+        all_params.append('_request_timeout')
+
+        params = locals()
+        for key, val in six.iteritems(params['kwargs']):
+            if key not in all_params:
+                raise TypeError(
+                    "Got an unexpected keyword argument '%s'"
+                    " to method get_sample_image_write_urls_by_id" % key
+                )
+            params[key] = val
+        del params['kwargs']
+        # verify the required parameter 'dataset_id' is set
+        if self.api_client.client_side_validation and ('dataset_id' not in params or
+                                                       params['dataset_id'] is None):  # noqa: E501
+            raise ValueError("Missing the required parameter `dataset_id` when calling `get_sample_image_write_urls_by_id`")  # noqa: E501
+        # verify the required parameter 'sample_id' is set
+        if self.api_client.client_side_validation and ('sample_id' not in params or
+                                                       params['sample_id'] is None):  # noqa: E501
+            raise ValueError("Missing the required parameter `sample_id` when calling `get_sample_image_write_urls_by_id`")  # noqa: E501
+
+        collection_formats = {}
+
+        path_params = {}
+        if 'dataset_id' in params:
+            path_params['datasetId'] = params['dataset_id']  # noqa: E501
+        if 'sample_id' in params:
+            path_params['sampleId'] = params['sample_id']  # noqa: E501
+
+        query_params = []
+
+        header_params = {}
+
+        form_params = []
+        local_var_files = {}
+
+        body_params = None
+        # HTTP header `Accept`
+        header_params['Accept'] = self.api_client.select_header_accept(
+            ['application/json'])  # noqa: E501
+
+        # Authentication setting
+        auth_settings = ['ApiKeyAuth', 'auth0Bearer']  # noqa: E501
+
+        return self.api_client.call_api(
+            '/v1/datasets/{datasetId}/samples/{sampleId}/writeurls', 'GET',
+            path_params,
+            query_params,
+            header_params,
+            body=body_params,
+            post_params=form_params,
+            files=local_var_files,
+            response_type='SampleWriteUrls',  # noqa: E501
```

Comment:
is this "normal" that we need to do this or did I mess up the openapi specs?

Reply:
This is autogenerated, thus I did not look at it. As long as it works, I'm happy.